docs/notifications.json

```diff
@@ -13,10 +13,5 @@
     "id": 55,
```

Comment:
Maybe we could bump the survey id in a couple of days. We are 4 days in and have 229 answers. Last year, 4 days in, we had 648 answers. I was expecting 900 answers this year at this point since we have +50% of the traffic on the website compared to last time. I'm not sure how to explain the difference. But I think that we need to figure this out before bumping the id. A couple of ideas: 1. the weekend 2. TypeForm is not the right tool for our survey (works well with 10 questions, not more) 3. the questions are too hard, too soon. 4. we miss an incentive. I think that it would be great to land on x10 the current number of answers. For instance, we have "How can we improve the Data Grid Pro for you?" 17 answers. I imagine that 170 would make a difference.